cmake/EthOptions.cmake

```diff
@@ -3,6 +3,7 @@ macro(configure_project)
 
 	# features
 	eth_default_option(COVERAGE OFF)
+	eth_default_option(OSSFUZZ OFF)
```

Comment:
Where is it set now?

Reply:
It is set in the ossfuzz specific solidity build script and will be part of an upcoming PR over at google/oss-fuzz.